department-of-veterans-affairs/va.gov-cms
# VACMS-2430: Remove seconds input from datetime fields.

## Description
See #2430 

## Testing done
Front end build / visual

## Screenshots
![image](https://user-images.githubusercontent.com/2404547/91213531-bced8680-e6df-11ea-922f-4ad310043b8e.png)

## QA steps
- As an admin, go to `node/6920/edit` and visually verify that seconds input does not appear in date widget.
- Change time, ensure node saves without errors.
